tests/test_callbacks.py

```diff
@@ -53,6 +81,20 @@ def test_enqueue_with_failure_callback(self):
         job = Job.fetch(id=job.id, connection=self.testconn)
         self.assertEqual(job.failure_callback, print)
 
+    def test_enqueue_with_stopped_callback_as_string(self):
```

Comment:
Hi, can you consolidate these three tests into one single test case? Using the regular `queue.enqueue()` is also sufficient.